airflow/contrib/example_dags/example_emr_job_flow_manual_steps.py

```diff
@@ -16,7 +16,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+"""
+This is an example dag for a AWS EMR Pipeline.
 
+Starting by creating a cluster,
```

Comment:
Personally, I don't like lists in file headers. It is very difficult to keep them up to date. It also seems to me that this sentence has formatting problems. If this is a list then each point should start with a star character.  If this is not a list then the sentence should be on one line

Reply:
This should not be a list. I split it into multiple lines because otherwise it would exceed the max line limit of 100.

Reply:
@mik-laj So do you think it is then better to put it in one line and have a pylint/flake8 disable/noqa comment?

Reply:
I think it's best to wrap the text.  I have a ruler on my IDE.  if the word is too long it gives enter before it.

Reply:
look at the random RFC document.  This is an example of how to format long lines.

Reply:
Hmm, okay. I will format it like you said - personally I like it to wrap it semantically on "pauses".